docs/developer/golang_version_check

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/sh
```

Comment:
If it's gonna be a script.. it better be .sh

Reply:
ya.. fine w/ that.  Teams go either way.. if it's executable and #! <something> it's not needed.  
happy to add